### PR TITLE
Specify the target Android SDK version.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/436331728536935

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://88-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://88-bb-docs.surge.sh

File: quickstart/android/integrate_sdk.adoc [hunk 1]
```diff
@@ -8,12 +8,32 @@ graphical feedback reporter, a crash reporting and analysis tool,
 automatic app updating and usage tracking. The SDK is regularly updated
 with new features.
 
-[WARNING]
-=========
+[IMPORTANT]
+===========
 **What's the minimum API version supported?**
 
 android:minSdkVersion="8"
```
Comment (kevin-bb) [inline quickstart/android/integrate_sdk.adoc]: `19`

**Comment by bbdocs** [comment]
See the website here: http://88-bb-docs.surge.sh

**Pull request merged**